__TESTS__/unit/actions/Resize.test.ts

```diff
@@ -4,7 +4,7 @@ import CloudinaryConfig from "../../../src/config/CloudinaryConfig";
 import {AutoGravity} from "../../../src/constants/gravityObjects/GravityObjects";
 import Gravity from "../../../src/params/gravity/Gravity";
 import * as GravityObjects from '../../../src/constants/gravityObjects/GravityObjects';
-import {IResizeAction} from "../../../src/actions/resize/IResizeAction";
+import {IResizeAction} from "../../../src/actions/resize/IResizeTypes";
```

Comment:
the descriptions says this was removed?

Reply:
The interface for ResizeAction was removed (initially we had an interface for every action, but it wasn't really needed).

We do have some Types left though that are needed. (If we decide to keep the resizeActionFactory)

Reply:
Good catch, removed.